src/main/java/com/epam/test_generator/entities/api/JwtTokenBuilder.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.test_generator.entities.api;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.JWTCreator;
+import com.auth0.jwt.algorithms.Algorithm;
+import com.epam.test_generator.entities.User;
+import org.springframework.core.env.Environment;
+
+import java.io.UnsupportedEncodingException;
+
+public class JwtTokenBuilder {
+    public JWTCreator.Builder buildFor(User user, String issuer) {
+        return JWT.create()
+                .withIssuer(issuer)
+                .withClaim("id", user.getId())
+                .withClaim("email", user.getEmail())
+                .withClaim("given_name", user.getName())
+                .withClaim("family_name", user.getSurname())
+                .withClaim("role", user.getRole().getName());
+    }
+
+    public String getLoginJWTToken(JWTCreator.Builder builder, Environment environment) {
```

Comment:
And this method is not getter, it's signer. I would rather name it *signJwtToken* or *signAuthenticationJwtToken*. Login cannot be used here because token used for authentication, not for login.